lego/apps/users/models.py

```diff
@@ -1,3 +1,4 @@
+# mypy: ignore-errors
```

Comment:
It's impossible to type to due a weird mypy bug, and I'm not gonna spend another month on this ...

Reply:
I'm guessing it's the `UserManager` that is causing problems? I spent some time on it as well, and it's a goddam nightmare, although I _believe_ it should be possible to fix. 

Do you have a reference to an issue report or similar? Would be nice to have that so we can check on it once in a while if this is fixed/simplified.